code/modules/antagonists/heretic/items/eldritch_painting.dm

```diff
@@ -0,0 +1,275 @@
+	name = "Great chaparral over rolling hills"
+	desc = "A painting depicting a massive thicket, it seems to be attempting to crawl through the frame. Destroyable with wirecutters."
+	icon_state = "eldritch_painting_vines"
+	sensor_type = null
+	// A static list of 5 pretty strong mutations, simple to expand for any admins
+	var/list/mutations = list(
+		/datum/spacevine_mutation/hardened,
+		/datum/spacevine_mutation/toxicity,
+		/datum/spacevine_mutation/thorns,
+		/datum/spacevine_mutation/fire_proof,
+		/datum/spacevine_mutation/aggressive_spread,
+		)
+	// Items to pick from to spawn on examine
+	var/list/items_to_spawn = list(
+		/obj/item/food/grown/poppy,
+		/obj/item/food/grown/harebell,
+	)
+
+/datum/mood_event/eldritch_painting/heretic_vines
+	description = "Oh what a lovely flower!"
+	mood_change = 3
+	timeout = 3 MINUTES
+
+/obj/structure/sign/painting/eldritch/vines/Initialize(mapload, dir, building)
+	new /datum/spacevine_controller(get_turf(src), mutations, 0, 10)
+	return ..()
+
+/obj/structure/sign/painting/eldritch/vines/examine_effects(mob/living/carbon/examiner)
+	. = ..()
+	if(IS_HERETIC(examiner))
+		var/item_to_spawn = pick(items_to_spawn)
+		to_chat(examiner, span_notice("You picture yourself in the thicket picking flowers.."))
+		// Spawns one of two flowers that often are used by heretics
+		new item_to_spawn(examiner.drop_location())
+		examiner.add_mood_event("heretic_vines", /datum/mood_event/eldritch_painting/heretic_vines)
+	else
+		new /datum/spacevine_controller(get_turf(examiner), mutations, 0, 10)
+		to_chat(examiner, span_notice("The thicket crawls through the frame, and you suddenly find vines beneath you..."))
+
+
+
+// Lady out of gates, gives a brain trauma causing the person to scratch themselves
+/obj/item/wallframe/painting/eldritch/beauty
+	name = "Lady out of gates"
+	desc = "A painting depicting a perfect lady, and I must be perfect like her..."
+	icon_state = "eldritch_painting_beauty"
+	result_path = /obj/structure/sign/painting/eldritch/beauty
+
+/obj/structure/sign/painting/eldritch/beauty
+	name = "Lady out of gates"
+	desc = "A painting depicting a perfect lady, and I must be perfect like her. Destroyable with wirecutters."
+	icon_state = "eldritch_painting_beauty"
+	sensor_type = /datum/proximity_monitor/advanced/eldritch_painting/beauty
+	// Set to mutadone by default to remove mutations
+	var/list/reagents_to_add = list(/datum/reagent/medicine/mutadone)
+
+// The special examine interaction for this painting
+/obj/structure/sign/painting/eldritch/beauty/examine_effects(mob/living/carbon/examiner)
+	. = ..()
+	if(IS_HERETIC(examiner))
+		to_chat(examiner, "Your imperfections shed and you are restored.")
+		examiner.reagents.add_reagent(reagents_to_add, 5)
+	if(!examiner.has_dna())
+		return
+	if(examiner.has_trauma_type(/datum/brain_trauma/severe/eldritch_beauty))
+		to_chat(examiner, "You feel changed, more perfect....")
+		examiner.easy_random_mutate(NEGATIVE + MINOR_NEGATIVE)
```

Comment:
is this supposed to do both if you're a heretic?

Reply:
If they have the trauma yes, though they shouldn't get the trauma in the first place unless its added manually by an admin

Reply:
Decided against this, just early returns on !is_heretic now